network-api/networkapi/templates/fragments/buyersguide/share-buttons.html

```diff
@@ -12,6 +12,13 @@
                           <span>{% trans "Facebook" %}</span>
                         {% endif %}
                     </a>
+                    <a class="tw-btn tw-btn-secondary btn-share after:tw-hidden mastodon-share" href="https://toot.kytta.dev/?text={{share_text|urlencode}} {{current_url|urlencode}}">
```

Comment:
I used the popular service named "toot" for a link to test the share link. Is it acceptable to use a third-party service? We need this kind of is needed because we cannot know the user's Mastodon domain name in advance.

You can read the details on this topic here:
https://github.com/mastodon/mastodon/issues/442#issuecomment-803916847